test-specifications/testSpecificationsTemplate.md

```diff
@@ -1,43 +1,59 @@
-# TestName - Test specification
+# RequestName - Test specification
 
 ## Description:
 
 ## Design:
 
-**Transaction properties:**
-
-**Crypto create protobufs:**
+**Request properties:**
 
+**<REQUEST_TYPE> protobufs:**
 
 **Response codes:**
 
+https://github.com/hashgraph/hedera-protobufs/blob/main/services/response_code.proto
+
+**Mirror Node APIs:**
+
+https://docs.hedera.com/hedera/sdks-and-apis/rest-api
+
 
 ## Initialisation:
 
 ```jsx
 
 ```
 
-## Properties
+## JSON-RPC API Endpoint Documentation
 
-### **PropertyName:**
+### Method Name
 
-- Description
+`<METHOD_NAME>`
+
+### Parameters
 
-| Test no | Name                                         | Input                                                         | Expected response                                                                                                                                               | Implemented (Y/N)
-| ------- | -------------------------------------------- | ------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |---------------------
-| 1       |                      |          |  |
-| 2       |         |                      |                                                                                                 |
-| 3       | |  |                                                                                                        |
+| Parameter Name     | Type   | Required/Optional | Description/Notes                                                                                                                           |
+|--------------------|--------|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+
+
+## Property/Function Tests
+
+### **Property/Function Name:**
+
+- Description
 
+| Test no | Name | Input | Expected response | Implemented (Y/N) |
+|---------|------|-------|-------------------|-------------------|
+| 1       |      |       |                   |                   |
+| 2       |      |       |                   |                   |
+| 3       |      |       |                   |                   |
 
-### **PropertyName:**
+### **Property/Function Name:**
```

Comment:
Duplicating to show that there can be multiple of them?

Reply:
That was the thought, but I have no issue with removing this if you think it's superfluous

Reply:
Understood. I think I am ok with it 🙂